tests/Data/ProgrammesDb/Util/SearchUtilitiesTraitTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+declare(strict_types = 1);
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\Util;
+
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\Util\SearchUtilitiesTrait;
+use PHPUnit\Framework\TestCase;
+
+class SearchUtilitiesTraitTest extends TestCase
+{
+    /** @var SearchUtilitiesTrait */
+    private $searchUtilityObject;
+
+    public function setUp()
+    {
+        $this->searchUtilityObject = $this->getObjectForTrait(SearchUtilitiesTrait::class);
+    }
+
+    /**
+     * @dataProvider stripPunctuationDataProvider
+     */
+    public function testStripPunctuation(string $string, string $expectedString)
```

Comment:
Looks like you could add the returning types to the functions in this class

Reply:
You mean that expectedString should be nullable? Or which return types are we talking about here?

Reply:
I mean this
```suggestion
    public function testStripPunctuation(string $string, string $expectedString): void
```

Reply:
ah, okay, we don't generally stick return type hints on our unit tests, I hadn't considered it. You think it's worth starting to do it?

Reply:
Could be a good practice for unitest as well but not something necesary because it has very little impact, so whatever you fancy